bitnami/supabase/templates/realtime/deployment.yaml

```
@@ -0,0 +1,181 @@
+{{- if .Values.realtime.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ template "supabase.realtime.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: supabase
+    app.kubernetes.io/component: realtime
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.realtime.replicaCount }}
+  {{- if .Values.realtime.updateStrategy }}
+  strategy: {{- toYaml .Values.realtime.updateStrategy | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: realtime
+  template:
+    metadata:
+      {{- if .Values.realtime.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.realtime.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: realtime
+        {{- if .Values.realtime.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.realtime.podLabels "context" $) | nindent 8 }}
+        {{- end }}
```

This is a recent change in out template:
```suggestion
        {{- end }}
        {{- if .Values.commonLabels }}
        {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 8 }}
        {{- end }}
```